### DISCUSSION TITLE
How to track the Maven version in a mvnvm.properties file?

### DISCUSSION THREAD

**Discussion by ablekhman** [renovatebot/renovate#12784] [Request Help]
We use `mvnvm` tool with our Maven-based Java project. `mvnvm` expects the `mvnvm.properties` file at the root directory of a project. Then it selects Maven version according to what specified in the file. The `mvnvm.properties` is a regular Java properties file with entries like this:
```text
mvn_version=3.8.1
```
Is it possible to configure Renovate to look into this file and suggest a new Maven version when it's available?

**Comment by viceice** [comment]
Yes, but currently only with a custom Regex manager

**Reply by ablekhman** [reply]
Do you have any example or documentation of how it can be done? How do I teach this Regex manager to look into `mvnvm.properties` or any other arbitrary file?

**Reply by viceice** [reply]
https://docs.renovatebot.com/modules/manager/regex/

**Comment by rarkins** [comment]
Feel free to create this as a feature request issue so that we can support it automatically in future. Please provide a minimal reproduction repo as part of the request.
